forc-plugins/forc-node/src/local/cmd.rs

```diff
@@ -0,0 +1,10 @@
+use clap::Parser;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug, Clone)]
+pub struct LocalCmd {
```

Comment:
Can we also have DB path for local node; it's good to be able to persist local node setup across restarts/runs

Reply:
This probably implies that `in-memory` is the default option (you may need to use an enum here so we can override to custom path)

Reply:
That is a good point! I think we should enable local node with persisting db but that shouldn't be the default. Maybe we can do it with a flag?

Reply:
Whatever is easiest/maintainable, don't really mind; thought an enum made sense since you'd either have 1 or the other option.
Another option is to use `Either` type (if we're certain there'd only ever be these 2 options); but an additional flag is fine too!